src/app/ApplicationTemplate.Mobile/Android/Resources/values/Styles.xml

```diff
@@ -1,17 +1,35 @@
-<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8" ?>
 <resources>
 	<style name="AppTheme" parent="Theme.AppCompat.Light">
-
 		<!-- This removes the ActionBar -->
 		<item name="windowActionBar">false</item>
 		<item name="android:windowActionBar">false</item>
 		<item name="windowNoTitle">true</item>
 		<item name="android:windowNoTitle">true</item>
-
 	</style>
 	<style name="Theme.AppCompat.Translucent">
```

Comment:
What does this do?

Reply:
Decided to remove it, I don't think it's very useful for now.

Reply:
I'm not sure to understand the contexte here? Did you have to add it, or it was there before?

Reply:
It was already there,

Reply:
oh ok, it's because you may have change the order which can of bring confusion on what changes

Reply:
The suggestion I would make here would be: if they are not any changes here (or minimal) I would try to keep the order so we can understand more what happen here.

Reply:
Verified and the order is like it was before.